x-pack/plugins/security_solution/public/detections/components/callouts/need_admin_for_update_callout/index.test.tsx

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { mount } from 'enzyme';
+import React from 'react';
+import { TestProviders } from '../../../../common/mock';
+import { NeedAdminForUpdateRulesCallOut } from '.';
```

Comment:
Ooops again 🙂

Reply:
I can change this one to be `./index` but usually for `index.ts` based files I just import from the folder directly vs something like `./index` and not find that a big issue with circles.

Reply:
Oh, I see, here it's not a big deal. Np 👍